src/Commands/Env/Update.php

```diff
@@ -10,6 +10,13 @@
 
 class Update extends BaseCommand
 {
+    /**
+     * @var array
+     */
+    protected $optionMap = [
```

Comment:
Not required when the keys are the same as the values.

Reply:
`fillData()` returns an empty array in this case...